cli/runtime/config/clientconfig_typemeta.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package config
+
+import (
+	"gopkg.in/yaml.v3"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/vmware-tanzu/tanzu-framework/cli/runtime/config/nodeutils"
+)
+
+//nolint:all
```

Comment:
nit: if possible remove nolint; otherwise provide the reason.

Reply:
added a reason to skip